docs/source/procedures/datadog/index.rst

```diff
@@ -48,7 +61,7 @@ Cloud users, skip this step, it's been take care for by the cloud.
 Upload the Dashboard
 ====================
 Download the dashboard file :download:`dashboard.json <dashboard.json>`.
-Create a new dashboard in Datadog and import the json file you downloaded. 
+Create a new dashboard in Datadog and import the json file you downloaded.
```

Comment:
dont we have a built in ScyllaDB dashboard on Datadog?

Reply:
We do, but it's not as good as this one, see #1831